PPOCRLabel/libs/canvas.py

```diff
@@ -593,7 +593,8 @@ def paintEvent(self, event):
             p.setPen(self.drawingRectColor)
             brush = QBrush(Qt.BDiagPattern)
             p.setBrush(brush)
-            p.drawRect(leftTop.x(), leftTop.y(), rectWidth, rectHeight)
+            rect = QRectF(leftTop.x(), leftTop.y(), rectWidth, rectHeight)
```

Comment:
Why we need `QRectF` Instead of `(x1, y1, w, h)`,  is it a problem with the `int` type?

Reply:
Yes ,Type conversion problem. 
Mac OS v13.6.6  ,Python 3.10.